src/main/java/org/folio/inventory/resources/TenantApi.java

```diff
@@ -24,11 +24,14 @@
 public class TenantApi {
   private static final Logger LOGGER = LogManager.getLogger(TenantApi.class);
 
+  private static final String CREATE_SCHEMA_SQL = "CREATE SCHEMA IF NOT EXISTS %s";
+  private static final String DROP_SCHEMA_SQL = "DROP SCHEMA IF EXISTS %1$s CASCADE; DROP ROLE IF EXISTS %1$s";
```

Comment:
Hello @AivarIUsupov, looks like we don't need to try to delete a role since we don't create it

Reply:
Hello @RuslanLavrov, removed it. Thank you!